source/designers-guide/lightweight-backend-modules/index.md

```diff
@@ -0,0 +1,627 @@
+---
+layout: default
+title: Lightweight backend modules
+github_link: designers-guide/lightweight-backend-modules/index.md
+shopware_version: 5.1.0
+indexed: true
+---
+
+## Introducing
+Shopware 5.1 comes with a pretty cool feature which is especially interesting for developers which would love to create a backend module for their plugin but don't want to deal with the steep learning curve of ExtJS. In the following article you'll learn anything about the new technology.
```

Comment:
http://englishplus.com/grammar/00000287.htm I think 'specially' is more adequate here